cmd/draupnir-finalise-image

```diff
@@ -111,7 +111,7 @@ for database in \$(psql -t -c "select datname from pg_database where not datname
 do
   echo ">> vacuuming \$database"
   psql --dbname "\$database" -t -c "select table_name from information_schema.tables where table_schema='public';" \
-    | xargs -P5 -n1 "$VACUUMDB" --dbname "\$database" --table
+    | xargs --no-run-if-empty -P5 -n1 "$VACUUMDB" --dbname "\$database"
```

Comment:
We can do better than this now that vacuumdb supports the jobs parameter.

Shall we update this PR or close it @benwh?

Reply:
Good point - done! Ready for another look